anthos-vmruntime/pos-vm.yaml

```diff
@@ -13,59 +13,36 @@
 # limitations under the License.
 
 # [START anthosbaremetal_anthos_vmruntime_pos_vm_datavolume_pos_vm_boot_dv]
-apiVersion: cdi.kubevirt.io/v1beta1
-kind: DataVolume
+apiVersion: vm.cluster.gke.io/v1
+kind: VirtualMachineDisk
 metadata:
-  creationTimestamp: null
   name: pos-vm-boot-dv
   namespace: default
 spec:
-  pvc:
-    accessModes:
-    - ReadWriteOnce
-    resources:
-      requests:
-        storage: 80Gi
-    storageClassName: local-shared
+  size: 80Gi
   source:
     http:
       url: https://storage.googleapis.com/pos-vm-images/pos-vm.qcow2
 # [END anthosbaremetal_anthos_vmruntime_pos_vm_datavolume_pos_vm_boot_dv]
 ---
 # [START anthosbaremetal_anthos_vmruntime_pos_vm_virtualmachine_pos_vm]
-apiVersion: vm.cluster.gke.io/v1alpha1
+apiVersion: vm.cluster.gke.io/v1
 kind: VirtualMachine
 metadata:
-  creationTimestamp: null
   name: pos-vm
   namespace: default
 spec:
-  compute: {}
-  disks: null
+  compute:
+    cpu:
+      vcpus: 2
+    memory:
+      capacity: 4Gi
+  disks:
+  - boot: true
+    driver: virtio
+    virtualMachineDiskName: pos-vm-boot-dv
   interfaces:
   - name: eth0
+    default: true
```

Comment:
There seems to be 2 minor differences to what was generated in my cluster:
https://paste.googleplex.com/5887058592661504

1. The `VirtualMachineDisk` has a `storageClassName: local-shared` attribute
2. The API for `VirtualMachine ` is listed as `vm.cluster.gke.io/vm.cluster.gke.io/v1` 

Not sure if the second is a bug. Can we use the paste above to ensure it is consistent with what is output?

Reply:
I'll add #1, and as discussed let's leave @2 as-is.